source/_integrations/ecoforest.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Ecoforest
+description: Instructions on how to integrate Ecoforest fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2023.10
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@pjanuario'
+ha_domain: ecoforest
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Ecoforest integration allows monitoring and control of local [Ecoforest](https://ecoforest.com) fireplaces in Home Assistant. These fireplaces are manufactured in Spain.
+
+There is currently support for the following device platforms within Home Assistant:
+
+- [Sensor](#sensor)
+
+{% include integrations/config_flow.md %}
+
+<div class='note'>
+Devices must first be connected to Wi-Fi or LAN using the app or website provided by the device manufacturer. It is not possible to connect devices to Wi-Fi using this integration.
```

Comment:
This sentence if odd / confusing:

`Devices must first be connected to Wi-Fi or LAN`

Which is next followed by:

`It is not possible to connect devices to Wi-Fi using this integration.`

So the first sentence is not correct / true?

Reply:
Changes requested are here https://github.com/home-assistant/home-assistant.io/pull/28975